lib/Source/io/newgrounds/NG.hx

```diff
@@ -238,8 +238,8 @@ class NG extends NGLite {
 		#elseif js
 			js.Browser.window.open(url, window);
 		#elseif desktop
-			
-			#if (sys && windows)
+
+			#if (sys && windows && hl)
```

Comment:
would this still work on non hashlink windows?

Reply:
did you try this on hashlink? If it didn't work before I don't see how this would fix anything, did you mean 
```hx
#if (sys && (windows || hl))
```

Reply:
It worked before, I reverted because it was depending of openFL
I tried on windows-hl, Idk If it works on different type of hashlink types, but I think it doesn't change right?

Reply:
I'm struggling to see how this change would work on windows builds that are NOT using hashlink, and if this change does work on hashlink, why was `&&hl` needed to begin with?

Reply:
Wait, I see where are you getting now,
I'm heckin stupid